src/CMakeLists.txt

```diff
@@ -442,6 +442,10 @@ if(HAVE_QATZIP)
   list(APPEND ceph_common_deps ${QATZIP_LIBRARIES})
 endif()
 
+if(HAVE_LIBCRYPTSETUP)
+  list(APPEND ceph_common_deps ${LIBCRYPTSETUP_LIBRARIES})
```

Comment:
Nit: can we just included this dependency in `librbd` as part of the `target_link_libraries`?